### ISSUE TITLE
Antivirus block Gradio, Help

### ISSUE THREAD

**Issue by KakaoXI** [RVC-Project/Retrieval-based-Voice-Conversion-WebUI#1116]
My antivirus updated yesterday and now when i open the webui, my antivirus blocks all the website and the banner, and idk what file i have to put in the excluions

**Comment by Tps-F** [comment]
https://github.com/gradio-app/gradio/issues/3230
I think this is a problem on the gradio side. Try adding it to the whitelist.
Also, it would be easier for us to advise you if you could tell us the specific OS and type of antivirus you have!

**Comment by KakaoXI** [comment]
> https://github.com/gradio-app/gradio/issues/3230
> 
> I think this is a problem on the gradio side. Try adding it to the whitelist.
> 
> Also, it would be easier for us to advise you if you could tell us the specific OS and type of antivirus you have!

The problem is that I put it in whitelist but nothing, I add the local host in whitelist, my antivirus is Kaspersky and I have windows 11 pro

**Comment by nzx3** [comment]
For me all gradio applications are giving me the Error, Connection time out after exactly 40 seconds. I noticed that the Google Colab was still running but the Gradio interface was with error. Even on the smartphone is doing the same thing, The only thing that they have in common is Kaspersky.

I just pause the protection and Gradio is not timing out after 40 seconds. As soon as the protection is running, every space that is using Gradio (Huggingface or Google Colab) is timing out with Error.

The easiest way to see it in action is with S[table diffusion 1.5 from Runwayml hugginface space](https://huggingface.co/spaces/runwayml/stable-diffusion-v1-5). Is generating the images and after 40 seconds the error is popping up. If you drag the place where the images are, you can see that the images are there.

So, it seems that the change Kaspersky did (who knows what that is), is cutting the connection after 40 seconds of waiting.
![Gradio Kaspersky Error](https://github.com/RVC-Project/Retrieval-based-Voice-Conversion-WebUI/assets/143946374/dd76918e-7a7d-4113-9ab6-3f5150610008)

I noticed for a couple of days that even the generated images on my local machine are not displayed anymore in Automatic1111 (Gradio).
I just paused the Kaspersky protection, and surprise surprise...the generated image was displayed.

It seems clearly that is between Kaspersky and Gradio.

I will try to raise the issue over there also.

(Windows 10, Free Kaspersky version)

**Comment by KakaoXI** [comment]
> For me all gradio applications are giving me the Error, Connection time out after exactly 40 seconds. I noticed that the Google Colab was still running but the Gradio interface was with error. Even on the smartphone is doing the same thing, The only thing that they have in common is Kaspersky.
> 
> I just pause the protection and Gradio is not timing out after 40 seconds. As soon as the protection is running, every space that is using Gradio (Huggingface or Google Colab) is timing out with Error.
> 
> The easiest way to see it in action is with S[table diffusion 1.5 from Runwayml hugginface space](https://huggingface.co/spaces/runwayml/stable-diffusion-v1-5). Is generating the images and after 40 seconds the error is popping up. If you drag the place where the images are, you can see that the images are there.
> 
> So, it seems that the change Kaspersky did (who knows what that is), is cutting the connection after 40 seconds of waiting. ![Gradio Kaspersky Error](https://user-images.githubusercontent.com/143946374/265268849-dd76918e-7a7d-4113-9ab6-3f5150610008.png)
> 
> I noticed for a couple of days that even the generated images on my local machine are not displayed anymore in Automatic1111 (Gradio). I just paused the Kaspersky protection, and surprise surprise...the generated image was displayed.
> 
> It seems clearly that is between Kaspersky and Gradio.
> 
> I will try to raise the issue over there also.
> 
> (Windows 10, Free Kaspersky version)

Yes same! I have also stable diffusion and says same error, i use also roop on stable and it says also cpuexecution provider and is slower, before was normal, but idk if is from kaspersky because also if i stop roop is slow same, but kaspersky always blocks gradio after the new kaspersky update, is true

**Comment by nzx3** [comment]
it seems that disabling the Safe Browsing Settings in Kaspersky, the Gradio connection will not be terminated after 40 seconds.

**Comment by KakaoXI** [comment]
> it seems that disabling the Safe Browsing Settings in Kaspersky, the Gradio connection will not be terminated after 40 seconds.

Yes i tried, i contacted kaspersky and i hope they can help about this
